pkg/actuators/machine/instances.go

```diff
@@ -310,6 +310,17 @@ func launchInstance(machine *machinev1.Machine, machineProviderConfig *machinev1
 		},
 	}
 
+	switch machineProviderConfig.NetworkInterfaceType {
+	case machinev1.AWSENANetworkInterfaceType:
+		networkInterfaces[0].InterfaceType = aws.String("interface")
+	case machinev1.AWSEFANetworkInterfaceType:
+		networkInterfaces[0].InterfaceType = aws.String("efa")
+	case "":
+		// Do nothing
```

Comment:
I'm not quite understand, why nothing here.
Would it make sense to put reasonable default there, just for make this behaviour clear (`aws.String("interface")` maybe).

Reply:
networkInterfaces[0].InterfaceType is pointer, I guess that default is nil

Reply:
Or just more detailed comment

Reply:
>networkInterfaces[0].InterfaceType is pointer, I guess that default is nil